package.json

```diff
@@ -54,6 +54,7 @@
   },
   "devDependencies": {
     "deep-equal": "~1.0.1",
+    "fh-forms": "1.16.7",
```

Comment:
Hi @davidkirwan,

Following the steps to update the dep here. 
* Run `npm install --save --save-exact fh-forms@1.16.7` to update the dep in the prod and not dev section
* Check if the dep was updated in the file [npm-shrinkwrap](https://github.com/feedhenry/fh-mbaas/blob/master/npm-shrinkwrap.json#L1181), if not delete:
** Delete the file and the node_modules dir 
** Run `npm install`
** Run `npm shrinkwrap` in the root dir of this project
* Then commit the changes 
* Also, add a new commit to bump the version of the fh-mbaas api.

Reply:
Ran the first command and ended up with this inside the npm-shrinkwrap.json file:

```
    "fh-forms": {
      "version": "1.16.6",
      "from": "fh-forms@1.16.6",
      "resolved": "https://registry.npmjs.org/fh-forms/-/fh-forms-1.16.6.tgz",
```

Reply:
Will retry by deleting this npm-shrinkwrap.json file, and the node_modules then following on with the ```npm install``` and ```npm shrinkwrap```

Reply:
This appeared to have worked.

Reply:
This should be removed from the devDependencies section, given that it's in the dependencies section now.